package.json

```diff
@@ -36,21 +36,32 @@
     "url": "https://github.com/conventional-changelog/conventional-changelog.git"
   },
   "scripts": {
+    "clear": "pnpm -r --parallel --if-present clear && del ./coverage",
+    "build": "pnpm -r --parallel --if-present build",
     "lint": "eslint . --cache --cache-location ./node_modules/.cache/ --ignore-path .gitignore",
     "fix": "eslint . --fix --cache --cache-location ./node_modules/.cache/ --ignore-path .gitignore",
     "test:watch": "vitest watch",
     "test": "vitest run --coverage"
   },
   "version": "1.0.0",
   "devDependencies": {
+    "@trigen/eslint-config": "8.0.0-alpha.27",
+    "@types/node": "^20.5.9",
     "@vitest/coverage-v8": "^0.34.0",
     "better-than-before": "^1.0.0",
     "conventional-changelog-core": "workspace:^",
+    "del-cli": "^5.1.0",
     "eslint": "^8.0.0",
     "eslint-config-standard": "^17.0.0",
-    "forceable-tty": "^0.1.0",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-n": "^16.0.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^6.0.0",
+    "eslint-plugin-standard": "^4.1.0",
     "safe-buffer": "5.2.1",
     "tmp": "^0.2.1",
+    "tsm": "^2.3.0",
```

Comment:
i'm surprised that you decided to go with tsm, what are the advantages compared to `ts-node/esm`

https://github.com/TypeStrong/ts-node?tab=readme-ov-file#node-flags-and-other-tools

Reply:
https://packagephobia.com/result?p=tsm@2.3.0
https://packagephobia.com/result?p=tsx@3.12.10
https://packagephobia.com/result?p=ts-node@10.9.1

tsx is larger
ts-node lighter, but in our case it should allow transpiling ts files from node_modules. I couldn't configure it to transpile it, tsm works great.